src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,35 +36,44 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={() => onClick(null)}
```

Comment:
From the name `onClick(null)` it is not very clear what will happen after the click.